src/sonic-yang-mgmt/sonic_yang.py

```diff
@@ -150,8 +163,7 @@ def print_data_mem (self, option):
         else:
             mem = self.root.print_mem(ly.LYD_XML, ly.LYP_WITHSIBLINGS | ly.LYP_FORMAT)
 
-        print("======================= print data =================")
-        print(mem)
+        return mem
```

Comment:
Intended to use this API to print out data tree.   But it is ok to just return the tree and caller can simply print the output.

Reply:
This API is called twice with "config breakout" and once with "config reload\load". Netops ppl will be baffled by too many lines of output, which they do not understand much. So We will print only the necessary messages while running config commands.